doc/tools/opensda.rst

```diff
@@ -107,8 +107,15 @@ application binary. :ref:`nxp_opensda_firmware` with this application.
 Flashing
 ========
 
-The Segger J-Link firmware does not support command line flashing, therefore
-the CMake ``flash`` target is not supported.
+Use the CMake ``flash`` target with the argument ``OPENSDA_FW=jlink`` to build
+your Zephyr application, invoke the J-Link Commander and program your Zephyr
+application to flash. Some boards set the argument by default, so it is not
```

Comment:
Since you don't know which boards set this argument, it might be better to delete this ``Some boards...`` sentence.